### PR TITLE
Include more items to meeting guidelines to be more inclusive

### PR DESCRIPTION
**Pull request by clarissalimab**
Some suggestions to the meeting guidelines to help be more inclusive.

Thanks, @gerondawollspill, for helping me phrase the power dynamics one!

### CODE REVIEW

**Review by MatheusRich — APPROVED** [review]
Love it.

File: working-together/README.md [hunk 1]
```diff
@@ -93,6 +97,12 @@ You can [download a double-sided printout] of these guides to display in your me
     - Always try to be explicit rather than implicit with regards to expressing emotions or intent,
       especially in text-based communication (where emojis and tone tags can be helpful).
       When others' intent is not explicit, assume positive or neutral intent instead of negative intent.
+- **Be mindful of power dynamics:** Be aware of power dynamics that may be
+  present in the meeting, such as those based on race, gender, or seniority.
+  Ensure that all participants have an opportunity to share their perspectives,
+  respectfully disagree, and that their contributions are valued equally.
+  - If you are in a position of power, consider speaking last to ensure that
```
Comment (minaslater) [inline working-together/README.md:104]: This is a great addition. Thanks for adding it.

**Review by crackofdusk — APPROVED** [review]

**Review by minaslater — APPROVED** [review]

**Review by thoughtbot-summer — APPROVED** [review]
This is wonderful. :sparkling_heart:

File: working-together/README.md (hunk 1 above)
Comment (gerondawollspill) [reply working-together/README.md:104]: This looks good to me! Great job @clarissalimab!

**Review by gerondawollspill — APPROVED** [review]

**Review by crystal-williams-brown — APPROVED** [review]
Great changes!

**Review by sarahraqueld — APPROVED** [review]
Love it, thanks @clarissalimab!

**Review by smaboshe — APPROVED** [review]

**Pull request merged**